### PR TITLE
RM-78

### PR DESCRIPTION
**Pull request by imtithalhalalo**
adding unpost for outbound transportation

### CODE REVIEW

**Comment by FatimaSalami** [comment]
image
this is a processed record, the icon should be locked
image
when unposted it is opened lock not the old img shape
(the old image shape should be used in screens were we don't have unpost)

**Comment by FatimaSalami** [comment]
i suggest not to let the unpost confirmation be expandable

File: src/components/Shared/StrictUnpostConfirmation.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import WindowToolbar from './WindowToolbar'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { Grid } from '@mui/material'
+
+const StrictUnpostConfirmation = ({ window, action }) => {
+  const [confirmationText, setConfirmationText] = useState('')
+
+  const { labels: _labels } = useResourceQuery({
+    datasetId: ResourceIds.SmsTemplates
```
Comment (Ghinwa-Arab) [inline src/components/Shared/StrictUnpostConfirmation.js:12]: what the purpose of passing smsTemplate resourceId??

**Comment by Ghinwa-Arab** [comment]
image

open any raw record, post button is enabled it should be disabled

**Comment by Ghinwa-Arab** [comment]
image

try to unpost then reopen , post button still enable 
once the record is not closed anymore post button should be disbaled

**Comment by Ghinwa-Arab** [comment]
image

new mode, post button enabled

**Comment by Ghinwa-Arab** [comment]
image

save new record, then click close

**Review by FatimaSalami — APPROVED** [review]

**Pull request merged**